src/averageAges.js

```diff
@@ -15,6 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const men = people.filter(person =>
+    century ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+      : person.sex === 'm');
+
+  const age = men.map(person => (person.died - person.born));
+
+  const agesSum = age.reduce((sum, x) =>
+    sum + x, 0);
+
+  const averageAge = agesSum / men.length;
```

Comment:
would be great to move this logic to a separate function and reuse in the next function